JustSaying/JustSayingBus.cs

```diff
@@ -158,33 +161,42 @@ private IMessagePublisher GetActivePublisherForMessage(Message message)
                 throw new InvalidOperationException(errorMessage);
             }
 
-            string activeRegion;
-            if (Config.GetActiveRegion == null)
+            var activeRegion = GetActiveRegion();
+            if (!string.Equals(_previousRegion, activeRegion, StringComparison.Ordinal))
```

Comment:
Exclude `null` from the logging here? Is it useful to log on the first execution that it changed from `null`?

Reply:
It's useful to log what it starts off as. The other option is to log the `null->some` transition as a special message.